### ISSUE TITLE
[Feature request] Gps altitude correction via elevation data and altitude filter 

### ISSUE THREAD

**Issue by neatwheat** [osmandapp/Osmand#8898]
I think that an improvement in gps/altitude "acurracy" could really push Osmand in terms of being the best offroad hiking alternative. I know that's a lot of wishful thinking, but nice to have would be

- an option for gps altitude correction with smrt/elevation data
- (an option to use only elevation data) 
- an gps filter for smoothing gps altitude tracking
- direct support for bluetooth gps

Concerning elevation data in general, it would be great if you could consider adding LIDAR data for Europe in the future as it is availaible here https://data.opendataportal.at/dataset/dtm-europe for a lot of countries.

**Comment by mahdi1234** [comment]
Yep, I was thinking like for two years log this enhancement request, would be great, just an example did hike yesterday which I know was like 700m but got total ascent over 3k :)

**Comment by sonora** [comment]
My opinion: If you really want to help this along, you should break it up into different issues or feature requests, and explain the envisioned use cases each. There is too many different fragments of separate ideas and user stories mentioned above. ;)

**Comment by vshcherb** [comment]
BTW We already use https://data.opendataportal.at/dataset/dtm-europe for SRTM / Slopes / Hillshades
